pkg/apis/nmstate/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -64,65 +64,6 @@ func (in ConditionList) DeepCopy() ConditionList {
 	return *out
 }
 
-// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
```

Comment:
Don't we have to maintain this, so we can still manage old versions ?

Reply:
For the operator, I'd rather assume there is nobody upgrading from the previous. For the handler, we will need to kepe backwards compatibility. Here it is not that important (IMO).

Reply:
fair enough